pkg/policy/rules.go

```diff
@@ -252,3 +253,29 @@ egressLoop:
 
 	return egressDecision
 }
+
+// analyzeWhetherRulesSelectEndpoints iterates over a given list of rules to
+// update the cache within the rule which determines whether or not the given
+// identity is selected by that rule. If a rule in the list does select said
+// identity, it is added to epIDSet. Signals to the given WaitGroup that
+// all rules have been parsed in relation to said identity.
+func (rules ruleSlice) analyzeWhetherRulesSelectEndpoint(identityConsumer IdentityConsumer, epIDSet *IDSet, wg *sync.WaitGroup) {
+	if identityConsumer == nil {
```

Comment:
Do you ever hit this case? My past experience of golang interfaces is that unless this function is directly called with `nil` as the parameter, then checking the argument against nil always fails (ie you wouldn't trigger this condition even if it's nil).

Reply:
I did indeed hit this case!